nucliadb/src/nucliadb/ingest/consumer/auditing.py

```diff
@@ -118,8 +119,18 @@ async def process_kb(self, kbid: str) -> None:
             total_fields += shard.fields
             total_paragraphs += shard.paragraphs
 
-        self.audit.report_fields_and_paragraphs(
-            kbid=kbid, paragraphs=total_paragraphs, fields=total_fields
+        async with datamanagers.with_ro_transaction() as txn:
+            num_vectorsets = (
+                len([vs async for vs in datamanagers.vectorsets.iter(txn=txn, kbid=kbid)]) or 1
+            )
+
+        self.audit.report_storage(
+            kbid=kbid,
+            paragraphs=total_paragraphs,
+            fields=total_fields,
+            bytes=total_paragraphs  # This is an estimation of bytes stored in a KB
+            * 10000
```

Comment:
use the `AVG_PARAGRAPH_SIZE_BYTES`  on `nucliadb/src/nucliadb/search/api/v1/knowledgebox.py` to keep this consistent with what nucliadb outputs of current size